observations/obs_converters/MPD/text_to_obs.f90

```diff
@@ -0,0 +1,206 @@
+! DART software - Copyright UCAR. This open source software is provided
+! by UCAR, "as is", without charge, subject to all terms of use at
+! http://www.image.ucar.edu/DAReS/DART/DART_download
+!
+! DART $Id$
+
+program text_to_obs
+
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+!
+!   text_to_obs - a program that only needs minor customization to read
+!      in a text-based dataset - either white-space separated values or
+!      fixed-width column data.
+!
+!     created 29 Mar 2010   nancy collins NCAR/IMAGe
+!     modified 20 Mar 2020  Michael Ying
+!
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+use         types_mod, only : r8, PI, DEG2RAD
+
+use     utilities_mod, only : initialize_utilities, finalize_utilities, &
+                              open_file, close_file
+
+use  time_manager_mod, only : time_type, set_calendar_type, set_date, &
+                              operator(>=), increment_time, get_time, &
+                              operator(-), GREGORIAN, operator(+), print_date
+
+use      location_mod, only : VERTISHEIGHT
+
+use  obs_sequence_mod, only : obs_sequence_type, obs_type, read_obs_seq, &
+                              static_init_obs_sequence, init_obs, write_obs_seq, &
+                              init_obs_sequence, get_num_obs, &
+                              set_copy_meta_data, set_qc_meta_data
+
+use obs_utilities_mod, only : create_3d_obs, add_obs_to_seq
+
+use      obs_kind_mod, only : MPD_ABSOLUTE_HUMIDITY
+
+implicit none
+
+character(len=64), parameter :: text_input_file = 'text.txt'
+character(len=64), parameter :: obs_out_file    = 'obs_seq.out'
```

Comment:
FYI:  character parameters can be specified with (len=*)
This is really compact and the variables never need to be 'trimmed' when being printed.